src/transform/FooterMenu.js

```diff
@@ -0,0 +1,219 @@
+import './FooterMenu.css'
+import Polyfill from './Polyfill'
+
+/**
+ * @typedef {function} FooterMenuItemPayloadExtractor
+ * @param {!Document} document
+ * @return {!Array.<string>} Important - should return empty array if no payload strings.
+ */
+
+/**
+ * @typedef {function} FooterMenuItemClickCallback
+ * @param {!Array.<string>} payload Important - should return empty array if no payload strings.
+ * @return {void}
+ */
+
+/**
+ * @typedef {number} MenuItemType
+ */
+
+// eslint-disable-next-line
+/**
+ * Extracts array of no-html page issues strings from document.
+ * @type {FooterMenuItemPayloadExtractor}
+ */
+const pageIssuesStringsArray = document => {
+  const tables = Polyfill.querySelectorAll(document,
+    'div#content_block_0 table.ambox:not(.ambox-multiple_issues):not(.ambox-notice)'
+  )
+  // Get the tables into a fragment so we can remove some elements without triggering a layout
+  const fragment = document.createDocumentFragment()
+  for (let i = 0; i < tables.length; i++) {
+    fragment.appendChild(tables[i].cloneNode(true))
+  }
+  // Remove some element so their text doesn't appear when we use "innerText"
```

Comment:
iirc I tried that. I'll add some tests with the tricky cases I found then we can safely try alternatives.